src/components/Dashboard/FundingCycles.tsx

```diff
@@ -95,18 +95,21 @@ export default function FundingCycles({
           text={t`Funding cycle`}
           tip={t`A project's lifetime is defined in funding cycles. If a funding target is set, the project can withdraw no more than the target for the duration of the cycle.`}
           style={{
-            marginBottom: 12,
+            marginBottom: 30,
```

Comment:
i think we talked about this in the Discord some but it may have gotten lost. this margin feels a bit too big to me, especially when the button isn't visible (for non-project owners). and for 99% of folks this button won't ever be on screen (note: in local development buttons like these are always visible for testing purposes, maybe we should change that)